libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/AdasisConfig.kt

```diff
@@ -0,0 +1,64 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Configuration of ADASISv2 feature.
+ *
+ * @param cycleTimes message generation cycle times configuration
+ * @param dataSending data sending configuration
+ * @param pathsOptions ADASISv2 path level specific configurations
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class AdasisConfig(
```

Comment:
internal constructor + builder
Otherwise it'll be more difficult to extend the API.
But probably also not critical for the snapshot.